src/Admin/Users.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Jilt for WooCommerce Promotions
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@skyverge.com so we can send you a copy immediately.
+ *
+ * @author    SkyVerge
+ * @copyright Copyright (c) 2020, SkyVerge, Inc. (info@skyverge.com)
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
+ */
+
+namespace SkyVerge\WooCommerce\Jilt_Promotions\Admin;
+
+use SkyVerge\WooCommerce\Jilt_Promotions\Handlers\Installation;
+use SkyVerge\WooCommerce\Jilt_Promotions\Handlers\Prompt;
+use SkyVerge\WooCommerce\Jilt_Promotions\Messages;
+use SkyVerge\WooCommerce\Jilt_Promotions\Notices\Notice;
+
+defined( 'ABSPATH' ) or exit;
+
+/**
+ * Handler for the notice shown when the merchant filters users by the Customer role.
+ *
+ * @since 1.1.0-dev.1
+ */
+class Users extends Prompt {
+
+
+	/** @var string the id associated with the message */
+	private $customer_role_message_id = 'users-customers-role';
+
+	/** @var string user page identifier */
+	private $users_screen_id = 'users';
+
+	/** @var string customer role to match with role parameter */
+	private $customer_role = 'customer';
+
+
+	/**
+	 * Renders a Notice object if the users customer role message is enabled.
+	 *
+	 * @internal
+	 *
+	 * @since 1.1.0-dev.1
+	 */
+	public function add_admin_notices() {
+
+		$notice = new Notice();
+		$notice->set_message_id( $this->customer_role_message_id );
+		$notice->set_actions( [
+			'label'   => __( 'Email my customers', 'sv-wc-jilt-promotions' ),
+			'name'    => 'email-my-customers-cta',
+			'url'     => 'https://www.skyverge.com/go/email-customers',
+			'primary' => true,
+			'type'    => 'link,'
+		] );
```

Comment:
@adrianostanley we need to define an array of options here. The `set_actions()` method can handle an array of arrays now: https://github.com/skyverge/wc-jilt-promotions/commit/82b16e2d737973e3128fabcf52146bb514b32ca1.

```suggestion
		$notice->set_actions( [
			[
				'name'    => 'email-my-customers-learn-more',
				'label'   => __( 'Learn more', 'sv-wc-jilt-promotions' ),
				'url'     => 'https://www.skyverge.com/go/email-customers',
				'type'    => Notice::ACTION_TYPE_LINK
			],
			[
				'name'    => 'email-my-customers-cta',
				'label'   => __( 'Email my customers', 'sv-wc-jilt-promotions' ),
				'primary' => true,
				'type'    => Notice::ACTION_TYPE_BUTTON
			]
	   ] );
```

Reply:
Sure thing.